BinaryEncoding.md

```diff
@@ -3,120 +3,143 @@
 This document describes the [portable](Portability.md) binary encoding of the
 [Abstract Syntax Tree](AstSemantics.md) nodes.
 
-The binary encoding is designed to allow fast startup, which includes reducing
-download size and allow for quick decoding. For more information, see the 
-[rationale document](Rationale.md#why-a-binary-encoding)
-
-Reducing download size, is achieved through three layers:
-
- * The **raw** binary encoding itself, natively decoded by the browser, and to
-   be standardized in the [MVP](MVP.md).
- * **Specific** compression to the binary encoding, that is unreasonable to
-   expect a generic compression algorithm like gzip to achieve.
-   * This is not meant to be standardized, at least not initially, as it can be
```

Comment:
This seems a larger discussion. If browsers don't have a streaming API to accept the encoded wasm and there is not a suitable solution to caching (matters still unknown) then for practically reasons the initial implementation will need to integrate some data-organisation to help generic compression. Further, if essential compression technology is not standardized, and is potentially encumbered and proprietary, then this may not be good for the web.

Can we please split these decisions out from just landing the v8 encoding as the 'the current state of the wasm binary spec'. I agree with @lukewagner to not name it the 'v8 encoding' - G seem fine for the project to adopt.